eth/catalyst/api.go

```diff
@@ -148,7 +148,8 @@ func (api *consensusAPI) AssembleBlock(params assembleBlockParams) (*executableD
 			return nil, err
 		}
 		parentL1BaseFee := fees.GetL1BaseFee(stateDb)
-		header.BaseFee = misc.CalcBaseFee(config, parent.Header(), parentL1BaseFee)
+		// Use time.Now() as the current block time to calculate the pending base fee.
```

Comment:
It's not `time.Now()`

Reply:
Removed in [13b13f9](https://github.com/scroll-tech/go-ethereum/pull/1199/commits/13b13f9cc948124a0a2781e5649875aff228f943).